AdvLoggerPkg/Library/AdvancedLoggerLib/PeiCore/AdvancedLoggerLib.c

```diff
@@ -623,17 +618,7 @@ AdvancedLoggerGetLoggerInfo (
       Status = PeiServicesInstallPpi (mAdvancedLoggerPpiList);
       ASSERT_EFI_ERROR (Status);
 
-      if (FeaturePcdGet (PcdAdvancedLoggerPeiInRAM)) {
-        LoggerInfo->InPermanentRAM = TRUE;
-        Status                     = MmUnblockMemoryRequest (NewLoggerInfo, Pages);

```

Comment:
Was this the only use of MmUnblockMemoryRequest? If so, we can get rid of that header and unlink the lib in the inf.

Reply:
This function call is used one other time in this file at around line # 240. So it looks like we will need to keep it.